app/javascript/mastodon/components/attachment_list.tsx

```diff
@@ -0,0 +1,55 @@
+import { FormattedMessage } from 'react-intl';
+
+import classNames from 'classnames';
+
+import ImmutablePureComponent from 'react-immutable-pure-component';
+
+import type { MediaAttachment } from 'app/javascript/types/media_attachments';
+import { Icon } from 'mastodon/components/icon';
+
+const filename = (url: string) => new URL(url).pathname.split('/').pop();
+
+interface Props {
+  media: MediaAttachment[];
```

Comment:
Should this be a `ImmutableList<MediaAttachment>` to match the previous `propTypes`?